include/net/net_pkt.h

```diff
@@ -225,7 +225,7 @@ struct net_pkt {
 	uint16_t ipv6_prev_hdr_start;
```

Comment:
In commit `net: ipv6_fragment: store M-bit in addition to the offset`, could you add more details what the M-bit means so something like
`also inspect the M-bit (More Fragment) of all received fragments.`

Reply:
sure

Reply:
done